ocaml/typing/typedtree.mli

```diff
@@ -80,10 +80,28 @@ module Unique_barrier : sig
 
   (* Resolve the unique barrier once type-checking is complete. *)
   val resolve : t -> Mode.Uniqueness.Const.t
+
+  val print : Format.formatter -> t -> unit
 end
 
+(** A unique use annotates accesses to an allocation.
+    In the type checker we ensure that
+      actual_mode           <= expected_mode
+      unique_use.uniqueness == expected_mode.uniqueness
+      unique_use.linearity  == actual_mode.linearity
```

Comment:
Probably better to say:
```
      unique_use.uniqueness <= expected_mode.uniqueness
      unique_use.linearity  >= actual_mode.linearity
```
The inequality is good enough for soundness; and equality is for completeness. This is to follow the same style of the first line.

Reply:
Sounds good!